package.json

```diff
@@ -69,9 +69,9 @@
     "jest": "jest",
     "lint": "./node_modules/.bin/eslint .",
     "lintfix": "./node_modules/.bin/eslint . --fix",
-    "pretest": "npm run lint && npm run csslint",
+    "pretest": "",
     "pretestwin": "npm run lint && npm run csslint",
-    "test": "jest && cd test/functional && ./test.sh",
+    "test": "cd test/functional && ./test.sh",
```

Comment:
What are these for?

Reply:
The positioning seems to be fixed now otherwise 👍

Reply:
Oops, missed this out.

Reply:
Fixed!